homeworks/artur.kobyliatsky_Allocene/GamesProject/games.html

```diff
@@ -0,0 +1,329 @@
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="cod2__description">
+                                <p>
+                                    <b>Genre:</b> Shooter
+                                </p>
+                                <p>
+                                    <b>Platform:</b> PC, PS5, Xbox One
+                                </p>
+                                <p>
+                                    <b>Publisher:</b> Activision Blizzard
+                                </p>
+                                <p>
+                                    <b>Developer:</b> Infinity Ward
+                                </p>
+                                <p>
+                                    <b>Release date:</b> 28-10-2022
+                                </p>
+                            </div>
+                        </div>
+
+                        <div class="games__card--cod2">
+                            <div class="cod2__general">
+                                <div class="cod2__general--img">
+                                    <img src="./img/Call_of_Duty-_Modern_Warfare_II.jpeg" alt="COD MW2">
+                                </div>
+                                <div class="cod2__general--desc">
+                                    <h3>COD: MWII</h3>
+                                    <p>
+                                        Big changes come to the COD: MWII formula in this sequel...
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="cod2__description">
+                                <p>
+                                    <b>Genre:</b> Shooter
+                                </p>
+                                <p>
+                                    <b>Platform:</b> PC, PS5, Xbox One
+                                </p>
+                                <p>
+                                    <b>Publisher:</b> Activision Blizzard
+                                </p>
+                                <p>
+                                    <b>Developer:</b> Infinity Ward
+                                </p>
+                                <p>
+                                    <b>Release date:</b> 28-10-2022
+                                </p>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+            </div>
+        </main>
+        <footer class="footer">
+            <div class="footer__container">
+                <div class="footer__content">
+                    <div class="social-icon">
+                        <div class="icon__fb">
+                            <a href="https://www.linkedin.com/feed/" target="_blank" rel="noopener">
```

Comment:
It is a set of the same elements, isn't it? So use `ul`, please

Reply:
I don’t quite understand what the problem is, I won’t be very comfortable working with the UL, there’s too much work for me, both a link and a picture, and it’s all in the UL.

Reply:
Okay, I rewrote it to UL